webview/src/plots/components/comparisonTable/cell/ComparisonTableCell.tsx

```diff
@@ -17,6 +22,17 @@ export const ComparisonTableCell: React.FC<{
 }> = ({ path, plot, imgAlt, boundingBoxClasses }) => {
   const plotImg = plot.imgs[0]
 
+  const boundingBoxCoords = useSelector(
+    (state: PlotsState) => state.comparison.boundingBoxPlotCoords
+  )
+  const getCellBoundingBoxCoords = createSelector(
```

Comment:
Discussed with @sroy3 in Slack, and we can use [`createSelector`](https://redux-toolkit.js.org/api/createSelector) to reduce rerendering when the coords change.

Reply:
This selector should be created outside of the component and used with `useSelector`.

`const cellBoundingBoxCoordsSelector = createSelector((state: PlotsState) => state.comparison.boundingBoxPlotCoords....)`

`const cellBoundingBoxCoords = useSelector((state) => cellBoundingBoxCoordsSelector(state, plot, path))`

Reply:
Done!